ziti/cmd/edge/quickstart.go

```diff
@@ -42,6 +42,7 @@ import (
 type QuickstartOpts struct {
 	Username           string
 	Password           string
+	InitOnly           bool
```

Comment:
I am reluctant to turn this into a 'kitchen sink' type of command that allows one to update/configure everything. With that type of lens, I don't know who would want to "InitOnly" or why. What's the driver here, who would use this and why?

Reply:
A simple set of command-line options is sufficient for what I have in mind, as long as I can edit the generated configs, and other state files, and restart the quickstart. That's where the main use case for `docker compose up` comes in.

The same approach may have a lesser value for someone who's developing Ziti itself, so I added a `--build` option to support that.

The main value seems to be running a mostly-default, minimal, local Ziti network, so I can use it with whatever I'm building on the platform, like zrok.

Reply:
> who would want to "InitOnly"

Oh, I needed this to automate bringing the quickstart because there is no way to express the intent "initialize if not initialized and run." 

Without `--init-only`, I could only say "fail if already initialized, else initialize and run."

Reply:
for me, I'm not sold that this is a good enough idea. we'll need to discuss

Reply:
Resolving this one because I removed `--init-only` after talking it over with @dovholuknf. That means it's necessary to wait until the router has synchronized API sessions and the Docker project is simpler because now there's no init-only step, which was a separate init container.